src/ws/test-sshtransport.c

```diff
@@ -145,7 +145,11 @@ setup_transport (TestCase *tc,
                  gconstpointer data)
 {
   const TestFixture *fixture = data;
+#ifdef g_assert_nonnull
   g_assert_nonnull(fixture);
+#else
+  g_assert_false(fixture == NULL);
+#endif
```

Comment:
K. I'll update it in a minute.
On Feb 5, 2015 2:32 PM, "Stef Walter" <notifications@github.com> wrote:

> In src/ws/test-sshtransport.c
> <https://github.com/cockpit-project/cockpit/pull/1773#discussion_r24162685>
> :
>
> >    g_assert_nonnull(fixture);
> > +#else
> > +  g_assert_false(fixture == NULL);
> > +#endif
>
> Hmmm, good catch, lets just do g_assert (fixture != NULL);
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockpit-project/cockpit/pull/1773/files#r24162685>.
>